test/cli/test/simple/test.yaml

```diff
@@ -49,6 +53,9 @@ results:
     rule: less-equal-than
     resource: test-lifetime-fail
     kind: Pod
+<<<<<<< HEAD
```

Comment:
Merge issues.

Reply:
I missed this, have corrected this in the latest commit.